BridgeCareApp/VuejsApp/src/components/Inventory.vue

```diff
@@ -74,9 +74,11 @@
     import axios from "axios";
     //@ts-ignore
     import AppSpinner from "../shared/AppSpinner";
-    import {IAttribute, IAttributeYearlyValue, ISection, mockSections} from "@/models/section";
+    import {IAttribute, IAttributeYearlyValue, ISection} from "@/models/section";
```

Comment:
You can remove //@ts-ignore and use from "../shared/AppSpinner.vue"
I know I was using it before. But adding .vue extension at the end is the better way